pages/linux/pacman-query.md

```diff
@@ -6,32 +6,32 @@
 
 - List installed packages and versions:
```

Comment:
```suggestion
- [Q]uery the local package database and list installed packages and versions:
```
Take a look at the previous page refreshes and try to pass the meaning of the options to the mnemonics
https://github.com/tldr-pages/tldr/pull/14658/files

Reply:
I don't see these issues being resolved.

Reply:
Should be good, also checked sync which answered my question below.